webpack/entrypoints.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
+  screen: ["sumo/scss/screen.scss"],
   common: [
     "sumo/js/i18n.js",
     "underscore/underscore.js",
     "moment/moment.js",
-    "jquery/dist/jquery.min.js",
-    "jquery/jquery-migrate.js",
+    "jquery",
+    // "jquery/jquery-migrate.js",
```

Comment:
is this intended? shall we remove instead?

Reply:
For now, there's an issue to figure out what happened with it: https://github.com/mozilla/sumo-project/issues/881